prow/spyglass/podlogartifact_fetcher_test.go

```diff
@@ -51,7 +52,7 @@ func TestFetchArtifacts_Prow(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
-		artifact, err := goodFetcher.Artifact(context.Background(), tc.job, tc.buildID, maxSize)
+		artifact, err := goodFetcher.Artifact(context.Background(), fmt.Sprintf("%s/%s", tc.job, tc.buildID), "build-log.txt", maxSize)
```

Comment:
I'm guessing the fmt.Sprintf returns the something-something-"key", right? Is there a (helper) function in use somewhere that already has this functionality? (If not, should there be?)